bin/reth/src/node/events.rs

```diff
@@ -299,6 +299,18 @@ impl Eta {
         self.last_checkpoint = current;
         self.last_checkpoint_time = Some(Instant::now());
     }
+
+    /// Format ETA for a given stage.
+    ///
+    /// NOTE: Currently ETA is disabled for Headers and Bodies stages until we find better
+    /// heuristics for calculation.
+    fn fmt_for_stage(&self, stage: StageId) -> String {
```

Comment:
should we instead completely remove the `eta` field if it's known to be `unknown` (lol) all the time?